src/App.js

```diff
@@ -1,25 +1,15 @@
 import React from 'react';
-let todoList = [ {
-  id: 1,
-  title: "Complete Assignment",
-}, 
-{
-  id: 2,
-  title: "Clean Room",
-}];
+import AddTodoForm from './AddTodoForm';
+import TodoList from './TodoList';
 
 function App() {
   return (
     <div style={{ textAlign: 'center' }}>
       <header>
         <h1>Todo List</h1>
+        <AddTodoForm />
+        < TodoList />
```

Comment:
The header is usually used for intro text or navigation. Try moving the `<AddTodoForm/>` and `<TodoList />` under the header.